### PR TITLE
rest fixes

### PR DESCRIPTION
**Pull request by alekseytupichenkov**
Hi, really good repo and good start for project 👍 
I made some fixes and corrections
- Change REST `user_create` rout url from `/api/users` to `/api/signup`, it need for `anonymous: true` in `config/packages/security.yaml`, I think it's ok when we can register new user without logging 
- Remove annotation `@Security(name="Bearer")` in actions which we can run anonymously
- Fix some SWG annotations
- Add some tests

### CODE REVIEW

File: src/UI/Http/Rest/Controller/Auth/CheckController.php [hunk 1]
```diff
@@ -51,8 +50,6 @@
      *
      * @SWG\Tag(name="Auth")
      *
-     * @Security(name="Bearer")
```
Comment (jorge07) [inline src/UI/Http/Rest/Controller/Auth/CheckController.php]: Good catch

File: src/UI/Http/Rest/Controller/Event/GetEventsController.php [hunk 2]
```diff
@@ -36,13 +36,14 @@ class GetEventsController extends QueryController
...
-     *         @SWG\Property(property="email", type="string")
-     *     )
+     *     name="page",
+     *     type="integer",
+     *     in="path"
+     * )
+     * @SWG\Parameter(
+     *     name="limit",
+     *     type="integer",
+     *     in="path"
```
Comment (jorge07) [inline src/UI/Http/Rest/Controller/Event/GetEventsController.php]: 👍

File: src/UI/Http/Rest/Controller/User/SignUpController.php [hunk 3]
```diff
@@ -41,26 +38,24 @@
...
      * @SWG\Tag(name="User")
      *
-     * @Security(name="Bearer")
-     *
      * @throws \Assert\AssertionFailedException
      */
     public function __invoke(Request $request): JsonResponse
     {
-        $uuid = $request->get('uuid');
+        $uuid = $request->get('uuid') ?: Uuid::uuid4()->toString();
```
Comment (jorge07) [inline src/UI/Http/Rest/Controller/User/SignUpController.php]: If we delegate to the Api the uuid generation but we return a 201... How can the client know ID of the resource created?

Comment (alekseytupichenkov) [reply src/UI/Http/Rest/Controller/User/SignUpController.php]: Hm, for call `/api/auth_check` client don't need uuid, after login client can call `/user/{email}`

Comment (alekseytupichenkov) [reply src/UI/Http/Rest/Controller/User/SignUpController.php]: And also I can return uuid :)

Comment (alekseytupichenkov) [reply src/UI/Http/Rest/Controller/User/SignUpController.php]: What is your opinion?

Comment (jorge07) [reply src/UI/Http/Rest/Controller/User/SignUpController.php]: I'm planing to replace the `GET /user/{email}` endpoint for `POST /user/_search` as I don't like the email in the url.

My approach... delegate the uuid creation to the client, it simplify all the things

Comment (alekseytupichenkov) [reply src/UI/Http/Rest/Controller/User/SignUpController.php]: Done 👍 
And i'll think about using this approach in project...

Comment (jorge07) [reply src/UI/Http/Rest/Controller/User/SignUpController.php]: When working with rest and hypermedia, you can also return a 201 and add a location header with the route of the new resource

**Comment by jorge07** [comment]
I'll look at why coverage decreased and then merge this. LGTM

**Comment by jorge07** [comment]
Thanks @alekseytupichenkov 

Finally merged 🎉

**Pull request merged**